assets/dev/scss/editor/_flex-icons.scss

```diff
@@ -6,20 +6,20 @@
 		max-width: 1em;
 	}
 
-	--is-ltr: 1;
-	--is-rtl: 0;
+	@if 'rtl' == $direction {
+		--is-ltr: 0;
+		--is-rtl: 1;
+	} @else {
+		--is-ltr: 1;
+		--is-rtl: 0;
+	}
```

Comment:
@rami-elementor can these variables be initialized in the same place & way you are initializing `--direction-multiplier`? 
This way you will be able to avoid using `$direction`